shells/pipes-shell/BUILD

```diff
@@ -12,6 +12,7 @@ run_in_repo(
     deps = [
         ":android_deploy_bin",
         ":web_deploy_srcs",
+        "//:all_srcs",
```

Comment:
Is this intentional, do we want to add building everything to pipes-shell build?

Reply:
Somehow added from #4012